### PR TITLE
Elaborated specs a bit

### PR DESCRIPTION
**Pull request by janvanmansum**

### CODE REVIEW

**Comment by janvanmansum** [comment]
@rvanheest, I have elaborated the specs a bit. Let me know if there are any more issues that need clarification.

File: README.md [hunk 1]
```diff
@@ -42,17 +42,6 @@ ARGUMENTS
 ---------
 <!-- TODO replace this with output from command line --help -->
 
---deposits-dir | -d (optional)
```
Comment (rvanheest) [inline README.md:45]: Why remove these arguments? According to line 10 you still have these arguments...

Comment (janvanmansum) [inline README.md:45]: Okay, this should be copied from the result of running the application with `--help` on the command line. Some things need to change:
- I propose to rename `--deposits-dir` to `--output-deposits-dir` to make it a bit more specific that this is the target of the application
- `sip-dir` should be `multi-deposit-dir`

Comment (rvanheest) [inline README.md:45]: Fine, should we have a default option of the `deposits-dir`, like we have in the `springfield-inbox`? Not sure about that. Currently this `deposit-dir` is optional, so either we should have a fallback or we need to change this into a required field.

Comment (janvanmansum) [inline README.md:45]: It should be a required field, lest we have no target for our output.

**Pull request merged**